samples/user-interface/haptics/README.md

```diff
@@ -0,0 +1,115 @@
+Haptic Sampler
+===========================
+
+Android provides wide access to the device vibrator hardware so you can play different
+vibration effects and patterns from your app. 
+
+Haptics can enrich your app experience by attracting
+the user's attention when it’s required, providing useful feedback to their interactions, and
+augmenting audio-visual content with tactile sensations.
+
+The sample is built on top of [Jetpack Compose](https://developer.android.com/jetpack/compose) and requires a recent version of [Android Studio](https://developer.android.com/studio). 
+
+It uses the Gradle build system. To build this project, clone the repo and use the "gradlew build" command or use "Import Project" in Android Studio (***File > New > Import Project***).
```

Comment:
Remove this part. build instructions are at the main reame

Reply:
Removed